src/main/java/codechicken/nei/guihook/GuiContainerManager.java

```diff
@@ -181,15 +181,28 @@ public static void drawItem(int i, int j, ItemStack itemstack) {
         drawItem(i, j, itemstack, getFontRenderer(itemstack));
     }
 
+    public static void drawItem(int i, int j, ItemStack itemstack, boolean smallAmount) {
+        drawItem(i, j, itemstack, getFontRenderer(itemstack), smallAmount);
+    }
+
+    public static void drawItem(int i, int j, ItemStack itemstack, FontRenderer fontRenderer) {
+        drawItem(i, j, itemstack, fontRenderer, false);
+    }
+
     private static int modelviewDepth = -1;
     private static final HashSet<String> stackTraces = new HashSet<>();
 
-    public static void drawItem(int i, int j, ItemStack itemstack, FontRenderer fontRenderer) {
```

Comment:
can't change the parameters of a public method.  Add a backwards compat method that will call drawItems with smallAmount set to a default value.

Reply:
it exists in line 188

Reply:
Haha, missed that.  :+1: